app/services/features.ts

```diff
@@ -0,0 +1,120 @@
+import Service from '@ember/service';
+import { assert } from '@ember/debug';
+import config from 'frontend-burgernabije-besluitendatabank/config/environment';
+
+export default class FeaturesService extends Service {
+  static PREFIX = 'feature-';
+  #features: Record<string, boolean> = {};
+
+  constructor() {
+    super();
+    const queryParams = this.#getQueryParams();
+    if (queryParams.get('clear-feature-overrides') === 'true') {
+      this.#clearCookieFeatures();
+    }
+
+    const configFeatures = this.#getConfigFeatures();
+    const cookieFeatures = this.#getCookieFeatures();
+    const queryFeatures = this.#getQueryParamsFeatures(queryParams);
+    this.setup({
+      ...configFeatures,
+      ...cookieFeatures,
+      ...queryFeatures,
+    });
+
+    // save query params in cookie
+    if (Object.keys(queryFeatures).length > 0) {
+      this.#setCookieFeatures(queryFeatures);
+    }
+  }
+
+  setup(features: Record<string, boolean>) {
+    this.#features = { ...features };
+    console.log('Feature flags:', this.#features);
```

Comment:
```suggestion
```

Reply:
I put it to simplify exchanges with future testers ;)